### PR TITLE
  docs(operators): document post-migration activeCLETH=0 exit deadloc…

### LINKED ISSUE #443: [Pectra Phase 1] Post-migration deadlock: requestETHExits unfulfillable until first oracle report (activeCLETH = 0)
### Description

`_migrateOperators_V2_3()` (`OperatorsRegistry.1.sol:55-66`) initializes every newly-pushed operator with `activeCLETH: 0`:

```solidity
OperatorsV3.push(
    OperatorsV3.Operator({
        funded: operator.funded * DEPOSIT_SIZE,
        requestedExits: operator.requestedExits * DEPOSIT_SIZE,
        active: operator.active,
        name: operator.name,
        operator: operator.operator,
        activeCLETH: 0 // This is ok to set 0 here because it will be updated via the oracle report before it gets used
    })
);
```

But `requestETHExits` (`OperatorsRegistry.1.sol:282-285`) derives the per-operator allocation cap from `activeCLETH`:

```solidity
uint256 opPendingExits = opRequestedExits - OperatorsV3.getExitedETHAtIndex(operatorIndex);
uint256 available = operator.activeCLETH > opPendingExits ? operator.activeCLETH - opPendingExits : 0;
if (ethAmount > available) {
    revert ExitsRequestedExceedAvailableFundedAmount(operatorIndex, ethAmount, available);
}
```

With `activeCLETH == 0`, `available == 0`, and any non-zero `ethAmount` reverts.

Meanwhile, `initOperatorsRegistryV1_2` carries `CurrentETHExitsDemand` and `TotalETHExitsRequested` over from V2 (× 32 ETH), so demand is **non-zero immediately after migration** but the keeper cannot allocate any of it until the first quorum-validated oracle report calls `_reportCLETH(_report.activeCLETHPerOperator)` (`OracleManager.1.sol:440`).

### Failure mode

If the first post-upgrade oracle report is delayed (rogue oracle member, stalled `epochsToAssumedFinality`, rejection from `InvalidValidatorCountReport` / `ExitedETHArrayShrinking`), the protocol cannot exit any validators — including pending redemptions queued before the upgrade. Failure is silent: keeper retries get `ExitsRequestedExceedAvailableFundedAmount(_, _, 0)`.

The fork test `2.operatorsMigrationV2toV3.t.sol` exercises `incrementFundedETH`, `addOperator`, and `setOperatorStatus` against migrated state, but **does not** exercise `requestETHExits`, so this is not caught by CI.

### Recommended fix (one of)

1. **Preferred:** seed `activeCLETH` from `funded - exitedETH` in `_migrateOperators_V2_3`. Best estimate from V2; the oracle's first report overwrites it anyway.
2. Fall back to `funded - requestedExits` semantics in `requestETHExits` when `activeCLETH == 0` (sentinel-gated, V2-compatible).
3. Document in the upgrade runbook that a fresh oracle report MUST land immediately after `initOperatorsRegistryV1_2`, with a redemption-queue pause covering the gap.

In all three cases, add a fork-test scenario that calls `requestETHExits` against fresh-migration state with non-zero `CurrentETHExitsDemand`.

---
*Filed as part of the Pectra Phase 1 differential review. Severity: HIGH.*

### PR DESCRIPTION
**Pull request by mischat**
…k (#443)

  _migrateOperators_V2_3 initializes migrated operators with activeCLETH = 0,
  which is only populated by the first post-upgrade oracle report. Because
  requestETHExits caps each operator's allocation by activeCLETH, no validator
  exits can be requested between initOperatorsRegistryV1_2 and that first report —
  even though CurrentETHExitsDemand / TotalETHExitsRequested are carried over
  non-zero from V2.

  Per the agreed remediation (#443, option 3) this is handled operationally rather
  than in code, so document the constraint where future readers will see it:

  - Expand the activeCLETH:0 inline comment (which previously understated the assumption) to spell out the deadlock and the required deploy sequencing.
  - Add a @dev warning to the IOperatorsRegistryV1.initOperatorsRegistryV1_2 NatSpec: a fresh oracle report MUST land immediately after the initializer, with the redemption queue paused over the gap.

  Docs-only; no behavioural or storage change. The runbook step (pause queue ->
  init -> land report -> unpause) lives in the upgrade runbook.

  Closes #443

## Notice

- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/liquid-collective/liquid-collective-protocol/pulls) for the same update/change?
- [ ] Have you assigned this PR to yourself?
- [ ] Have you added at least 1 reviewer?
- [ ] Have you updated the official documentation?
- [ ] Have you added sufficient documentation in your code?
- [ ] Have you added relevant tests to the official test suite?

## Pull Request Type

- [ ] 💫 New Feature (Breaking Change)
- [ ] 💫 New Feature (Non-breaking Change)
- [ ] 🛠️ Bug fix (Non-breaking Change: Fixes an issue)
- [ ] 🕹️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)

## Testing

- [ ] Have you tested this code with the official test suite?
- [ ] Have you tested this code manually?

### CODE REVIEW

File: contracts/src/interfaces/IOperatorRegistry.1.sol [hunk 1]
```diff
@@ -277,6 +277,13 @@ interface IOperatorsRegistryV1 {
     // function initOperatorsRegistryV1_1() external;
 
     /// @notice Migrates operators from V2 to V3 storage, dropping key-management fields
+    /// @dev    Migrated operators start with `activeCLETH == 0`, which is only populated by the first
+    ///         post-upgrade oracle report. Since `requestETHExits` caps each operator's allocation by
+    ///         `activeCLETH`, NO validator exits can be requested between this call and that first
```
Comment (juliaaschmidt) [inline contracts/src/interfaces/IOperatorRegistry.1.sol]: nitpick
```suggestion
    ///         `activeCLETH`, no validator exits can be requested between this call and that first
```
NO is node operator

File: contracts/src/interfaces/IOperatorRegistry.1.sol [hunk 2]
```diff
@@ -277,6 +277,13 @@ interface IOperatorsRegistryV1 {
...
+    ///         report — even though `CurrentETHExitsDemand` / `TotalETHExitsRequested` are already
+    ///         non-zero (carried over from V2). OPERATIONAL REQUIREMENT: a fresh oracle report MUST
+    ///         land immediately after this call, and the redemption queue MUST be paused over the
+    ///         gap. See issue #443.
```
Comment (juliaaschmidt) [inline contracts/src/interfaces/IOperatorRegistry.1.sol]: do not mention issue in src code?

File: contracts/src/OperatorsRegistry.1.sol [hunk 3]
```diff
@@ -67,7 +67,14 @@ contract OperatorsRegistryV1 is IOperatorsRegistryV1, Initializable, Administrab
...
                     name: operatorV2.name,
                     operator: operatorV2.operator,
-                    activeCLETH: 0 // This is ok to set 0 here because it will be updated via the oracle report before it gets used
+                    // activeCLETH starts at 0 and is only populated by the first post-upgrade oracle
+                    // report (OracleManager._reportCLETH). UNTIL that report lands, requestETHExits caps
+                    // each operator's allocation by activeCLETH, so `available == 0` and any non-zero exit
+                    // allocation reverts with ExitsRequestedExceedAvailableFundedAmount — even though
+                    // CurrentETHExitsDemand / TotalETHExitsRequested are carried over non-zero from V2.
+                    // A fresh oracle report MUST land immediately after initOperatorsRegistryV1_2, with
+                    // the redemption queue paused to cover the gap. See issue #443.
```
Comment (juliaaschmidt) [inline contracts/src/OperatorsRegistry.1.sol]: remove issue reference

**Review by juliaaschmidt — DISMISSED** [review]

File: contracts/src/interfaces/IOperatorRegistry.1.sol [hunk 4]
```diff
@@ -277,6 +277,13 @@ interface IOperatorsRegistryV1 {
     // function initOperatorsRegistryV1_1() external;
 
     /// @notice Migrates operators from V2 to V3 storage, dropping key-management fields
+    /// @dev    Migrated operators start with `activeCLETH == 0`, which is only populated by the first
+    ///         post-upgrade oracle report. Since `requestETHExits` caps each operator's allocation by
+    ///         `activeCLETH`, no validator exits can be requested between this call and that first
+    ///         report — even though `CurrentETHExitsDemand` / `TotalETHExitsRequested` are already
+    ///         non-zero (carried over from V2). OPERATIONAL REQUIREMENT: a fresh oracle report MUST
+    ///         land immediately after this call, and the redemption queue MUST be paused over the
+    ///         gap. See issue #443.
```
Comment (mischat) [inline contracts/src/interfaces/IOperatorRegistry.1.sol]:
```suggestion
    ///         gap.
```

File: contracts/src/OperatorsRegistry.1.sol (hunk 3 above)
Comment (mischat) [inline contracts/src/OperatorsRegistry.1.sol]:
```suggestion
                    // the redemption queue paused to cover the gap.
```

**Review by juliaaschmidt — APPROVED** [review]
LGTM
